src/App.js

```diff
@@ -1,43 +1,44 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Routes, Route } from "react-router-dom";
+import {
+	Home,
+	Login,
+	Liked,
+	Playlist,
+	Signup,
+	History,
+	WatchLater,
+	UserProfile,
+} from "./frontend/pages";
+import { PageContainer } from "./frontend/components";
+import Mockman from "mockman-js";
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} alt="mockBee logo" width="180" height="180" />
-        <h1 className="brand-title">
-          Welcome to <span>mockBee!</span>
-        </h1>
-        <p className="brand-description">
-          Get started by editing <code>src/App.js</code>
-        </p>
-        <div className="links">
-          <a
-            href="https://mockbee.netlify.app/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Explore mockBee
-          </a>
-          <a
-            href="https://mockbee.netlify.app/docs/api/introduction"
-            target="_blank"
-            rel="noreferrer"
-          >
-            API Documentation
-          </a>
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
-    </div>
-  );
+	return (
+		<div className="App">
+			<Routes>
+				<Route path="/" element={<PageContainer page={<Home />} />} />
+				<Route path="/liked" element={<PageContainer page={<Liked />} />} />
+				<Route
+					path="/watchLater"
+					element={<PageContainer page={<WatchLater />} />}
+				/>
+				<Route
+					path="/playList"
+					element={<PageContainer page={<Playlist />} />}
+				/>
+				<Route path="/history" element={<PageContainer page={<History />} />} />
+				<Route path="/login" element={<Login />} />
+				<Route path="/signup" element={<Signup />} />
+				<Route
+					path="/userProfile"
+					element={<PageContainer page={<UserProfile />} />}
+				/>
```

Comment:
You can add a route with `path="*"` for 404 page

Reply:
yes Added